.github/workflows/develop-ci.yml

```
@@ -0,0 +1,105 @@
+    needs: test # test job이 끝나야 실행
+    runs-on: ubuntu-latest
+    if: always()  # 성공하든 실패하든
+    steps:
+      - name: Notify Discord
+        env:
+          DISCORD_WEBHOOK: ${{ secrets.DISCORD_WEBHOOK }}
+          STATUS: ${{ job.status }}
+        run: |
+          message="✅CI 성공"
+          color=3066993 # 초록
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)